examples/PhasorDynamics/Tiny/TwoBus/Tgov1/TwoBusTgov1.cpp

```diff
@@ -107,15 +107,16 @@ int main()
   data.genrou[0].parameters[GenrouParameters::S12]   = 0.;
 
   data.gov.resize(1);
-
-  // Set Gov data (Default PW values)
-  data.gov[0].R     = 0.05;
-  data.gov[0].Pvmin = 0;
-  data.gov[0].Pvmax = 1.0;
-  data.gov[0].T1    = 0.5;
-  data.gov[0].T2    = 2.5;
-  data.gov[0].T3    = 7.5;
-  data.gov[0].Dt    = 0;
+  using P = GridKit::PhasorDynamics::Governor::Tgov1Parameters;
+
+  auto& gov0                = data.gov[0];
+  gov0.parameters[P::R]     = 0.05;
+  gov0.parameters[P::Pvmin] = 0.0;
+  gov0.parameters[P::Pvmax] = 1.0;
+  gov0.parameters[P::T1]    = 0.5;
+  gov0.parameters[P::T2]    = 2.5;
+  gov0.parameters[P::T3]    = 7.5;
+  gov0.parameters[P::Dt]    = 0.0;
```

Comment:
i suggest stripping the using declaration for `P` to avoid cluttering the scope here and for consistency purposes. instead add a `using namespace GridKit::PhasorDynamics::Governor` and do `Tgov1Parameters::<parameter here>` for each. additionally, swap in `data.gov[0].` (again, for consistency).